src/AzureClient/Visualization/ResourceEstimationEncoder.cs

```diff
@@ -74,9 +109,9 @@ public record class ResourceEstimationToHtmlEncoder(ILogger? Logger = null) : IR
                     {
                         var value = result.GetValueFromPath(entry.GetValue<string>("path"));
                         sb.AppendLine($@"
-                            <tr title='{entry["description"]}'>
+                            <tr class=aqre-tooltip>
                                 <td>{entry["label"]}</td>
-                                <td>{value}</td>
+                                <td class=aqre-tooltip>{value}<span class=aqre-tooltiptext>{entry["description"]}</span></td>
```

Comment:
Like this the tooltip is only accessible via the second column. It would be better to have it on the whole row.

Reply:
When I tested this, it was accessible over the whole row!

Reply:
You're right, I saw it wrong.